fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/OverworldBiomes.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.biomes.v1;
+
+import net.fabricmc.fabric.impl.biomes.InternalBiomeData;
+import net.minecraft.world.biome.Biome;
+
+/**
+ * API that exposes some internals of the minecraft default biome source for the overworld
+ */
+public final class OverworldBiomes {
+
+	private OverworldBiomes() {
+	}
+
+	/**
+	 * Adds the biome to the specified climate group, with the specified weight
+	 *
+	 * @param biome the biome to be added
+	 * @param climate the climate group whereto the biome is added
+	 * @param weight the weight of the entry. The weight in this method corresponds to its selection likelihood, with
+	 * 	             heavier biomes being more likely to be selected and lighter biomes being selected with less likelihood.
+	 * 	             Vanilla weights for each biome in each climate are listed in the {@link OverworldClimate} javadoc.
+	 */
+	public static void addBaseBiome(Biome biome, OverworldClimate climate, double weight) {
+		InternalBiomeData.addOverworldBaseBiome(climate, biome, weight);
+	}
+
+	/**
+	 * Adds the biome as a hills variant of the parent biome, with the specified weight
+	 *
+	 * @param parent the biome to where the hills variant is added
+	 * @param hills the biome to be set as a hills variant
+	 * @param weight the weight of the entry. The weight in this method corresponds to its selection likelihood, with
+	 *               heavier biomes being more likely to be selected and lighter biomes being selected with less likelihood.
+	 *               Mods should use 1.0 as the default/normal weight.
```

Comment:
Introduce a constant `OverworldBiomes.DEFAULT_WEIGHT`
```java
public static final double DEFAULT_WEIGHT = 1.0;
```
```suggestion
	 * @see OverworldBiomes#DEFAULT_WEIGHT for the default biome weight
```

Reply:
eh I don't think so. Perhaps the wording was poor but it just means that most vanilla biomes would have a weight equivalent to 1, so if you want to match vanilla, you most likely just want 1 (but even in vanilla, some have weights of 3 and 2, as listed in OverworldClimate

Reply:
@Prospector this is for hills biomes as far as I can see